oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -273,4 +280,73 @@ private static long getFileDescriptors(int index) {
         }
         return 0L;
     }
+
+    /**
+     * Get logical volume groups on this machine
+     *
+     * Instantiates a list of {@link VolumeGroup} objects,
+     * representing a storage pool, device, partition, volume, concrete file system
+     * or other implementation specific means of file storage.
+     *
+     * @return A list of {@link VolumeGroup} objects or an empty
+     *      *         array if none are present.
+     */
+
+    public List<VolumeGroup> getLogicalVolumeGroups() {
+        Map<String, Map<String, List<String>>> logicalVolumesMap = new HashMap<>();
+        Map<String, Set<String>> physicalVolumesMap = new HashMap<>();
```

Comment:
I think there's value in having this as the same map setup as the logical volume map, e.g., quickly identifying "unused" physical volumes, and it doesn't cost us anything in the iteration to add to both maps when we find a linkage.